askama_shared/src/parser.rs

```diff
@@ -132,14 +132,16 @@ pub(crate) enum Target<'a> {
 #[derive(Clone, Copy, Debug, PartialEq)]
 pub(crate) enum Whitespace {
```

Comment:
Couldn't `enum WhitespaceHandling` be renamed into `enum Whitespace` and then reuse the same `enum Whitespace` for both things? Is there any real need to have two different types in this case? :)

Reply:
I don't like too much to mix parser and config although they are the same type in here. Well, if @djc agrees with you, I'll merge them. :)

Reply:
To be fair, I do agree with that point-of-view. I was just curious if there was anything beyond that :)

Reply:
No, just this "mental split". Nothing else. :smile:

Reply:
The main downside here is the ugly name of `WhitespaceHandling`, but I think it makes sense to keep them separate for now.